package.json

```diff
@@ -16,7 +16,7 @@
   "license": "ISC",
   "dependencies": {
     "heroku-cli-util": "^8.0.8",
-    "heroku-exec-util": "0.7.3",
+    "heroku-exec-util": "https://github.com/heroku/heroku-exec-util#start-checking-host-keys",
```

Comment:
could/should this be an ENV var with a default (or some other kind of param/arg)?

Reply:
Well this is just temporary until that branch of `heroku-exec-util` is merged and released. Once that's done I plan to come here and make it a normal update (to `0.7.4` or whatever). I just did this for now so the CI build (and anyone that wants to run it locally) gets the updated version. I guess it would be simpler to be able to override with an environment variable or something but not sure if that's possible..? I've never found it too bad doing it this way really.